frontend/__tests__/routes/git-settings.test.tsx

```diff
@@ -243,15 +244,30 @@ describe("Form submission", () => {
     expect(saveProvidersSpy).toHaveBeenCalledWith({
       github: { token: "test-token", host: "" },
       gitlab: { token: "", host: "" },
+      bitbucket: { token: "", host: "" },
```

Comment:
This test file should have tests that check whether github is settable, gitlab is settable, and bitbucket is settable, instead of testing just github and github+gitlab.